### PR TITLE
Bug ?: Update for bug bounty q1 2016

### PR DESCRIPTION
**Pull request by pmac**
Replaces #4138 with a rebase.

### CODE REVIEW

**Comment by pmac** [comment]
@rforbes Look okay? Do you know if there's a bug associated with this change? I'll add it to the commit message if so.

**Comment by rforbes** [comment]
looks good to me.  There is no bug.

File: bedrock/security/templates/security/bug-bounty/hall-of-fame.html [hunk 1]
```diff
@@ -32,6 +32,42 @@ <h1 itemprop="name" class="title-shadow-box">Firefox Bug Bounty Rewards</h1>
...
+            <li>Aki Helin</li>
+            <li>Alexander Klink</li>
+            <li>Chamal</li>
+            <li>Dominique Hazaël-Massieux</li>
+            <li>Frederic Besler @ LAF INTL</li>
+            <li>Frédéric Hoguin</li>
+            <li>Gustavo Grieco</li>
+            <li>Holger Fuhrmannek</li>
+            <li>James Clawson</li>
+            <li>jasonpang2011@gmail.com</li>
```
Comment (alexgibson) [inline bedrock/security/templates/security/bug-bounty/hall-of-fame.html]: Are we OK publishing someone's email address? Could we just use `Jason Pang`?

**Comment by rforbes** [comment]
That is an error, thank you or catching that.  Can we change that inline or will I need to make a new pull request? That should just be Jason Pang.

**Comment by pmac** [comment]
Fixed the email issue. Good to go now I think.

**Comment by alexgibson** [comment]
r+ - thannks @rforbes! (and @pmac)

**Pull request merged**